share/spack/gitlab/cloud_pipelines/stacks/data-vis-sdk/spack.yaml

```diff
@@ -18,7 +18,7 @@ spack:
     paraview:
       variants: +osmesa
     all:
-      target: [x86_64]
+      target: [x86_64_v3]
```

Comment:
is this target necessary if the target=x86_64_v3 is required on all packages below? Does this string actually impact the hash when concretizing?